openedx_lti_tool_plugin/templates/openedx_lti_tool_plugin/courseware.html

```diff
@@ -0,0 +1,37 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset='utf-8'>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+  </head>
+
+  <body>
+    <iframe
+      id='unit-iframe'
+      src='{{ xblock_url }}'
+      allow='microphone *; camera *; midi *; geolocation *; encrypted-media *'
+      allowfullscreen=''
+      scrolling='no'>
```

Comment:
```suggestion
      scrolling='no'
```

Reply:
Do you mean remove scrolling or the closing tag?, removing the tag breaks the iframe